### PR TITLE
{vis}[foss/2016a] PyGTS 0.3.1 (REVIEW)

### PR DESCRIPTION
**Pull request by jrbosch**
add PyGTS easyconfig it's a dependency needed by Yade 1.20.0

### CODE REVIEW

File: easybuild/easyconfigs/p/PyGTS/PyGTS-0.3.1-foss-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('GLib', '2.47.5'),
+    ('GTS' ,'0.7.6'),
+    ('numpy', '1.8.2', versionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyGTS/PyGTS-0.3.1-foss-2016a-Python-2.7.11.eb]: @jrbosch why the old numpy? There's a numpy already included in the Python dependency itself...

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
@jrbosch missing easyconfig for the dependency?

```
EasyBuildError: 'Irresolvable dependencies encountered: GTS/0.7.6-foss-2016a'
```

It's available in #2839, but we need it here (too)?

**Comment by jrbosch** [comment]
I will sync add_easyconfig_yade branch to resolve the merge conflicts because GTS after merging #2969 
thanks @boegel

File: easybuild/easyconfigs/g/GTS/GTS-0.7.6-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ It is an Open Source Free Software Library intended to provide a set of useful
+ functions to deal with 3D surfaces meshed with interconnected triangles."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('GLib', '2.47.5'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/GTS/GTS-0.7.6-foss-2016a.eb]: there's an update for GLib, v2.48.0, I'm working on an easyconfig for it, PR coming up...

File: easybuild/easyconfigs/p/PyGTS/PyGTS-0.3.1-foss-2016a-Python-2.7.11.eb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ It is a hand-crafted and pythonic binding for the GNU Triangulated Surface (GTS) Library. """
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('GLib', '2.47.5'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyGTS/PyGTS-0.3.1-foss-2016a-Python-2.7.11.eb]: bump to 2.48.0?

File: easybuild/easyconfigs/g/GTS/GTS-0.7.6-foss-2016a.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/g/GTS/GTS-0.7.6-foss-2016a.eb]: @jrbosch see #2973

**Comment by boegel** [comment]
@jrbosch #2973 which provides GLib 2.48.0 is merged

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2606.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/97239ca83a51920c6253619daacc04b4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6549437b4fe3929fc2929ecf36f7dc7d for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @jrbosch!

**Pull request merged**